test/model/create.test.ts

```diff
@@ -7,25 +7,40 @@ test('creates a new entity', () => {
   const db = factory({
     user: {
       id: primaryKey(identity(userId)),
+      name: () => 'Mark Chandler',
+      nicknames: () => ['with-heart'],
+      favoriteNumbers: () => [69, 420],
```

Comment:
I don't know that this is the best file to test this stuff in, but it's what made sense to me at the time. Can move/restructure as needed.

Reply:
You've chosen the correct test suite: I think various data types support for the model declaration should be listed in the `create` test suite 👌